spec/unit/configurer_spec.rb

```diff
@@ -765,4 +796,32 @@ def method_missing(*args)
       @agent.convert_catalog(@oldcatalog, 10)
     end
   end
+
+  describe "when determining whether to pluginsync" do
+    it "should default to Puppet[:pluginsync] when explicitly set by the commandline" do
+      Puppet.settings[:pluginsync] = false
```

Comment:
I'm not sure why, but sometimes we do `Puppet[:name]` and sometimes `Puppet.settings[:name]` in unit tests?